automation/taskcluster/lib/tasks.py

```diff
@@ -49,14 +49,18 @@ def __init__(
     def craft_assemble_nightly_task(self, variant, version_name, is_staging):
         artifacts = _craft_artifacts_from_variant(variant)
 
-        sentry_secret = '{}project/mobile/reference-browser/sentry'.format(
+        secret_index = '{}project/mobile/reference-browser'.format(
             'garbage/staging/' if is_staging else ''
         )
 
         pre_gradle_commands = (
             'python automation/taskcluster/helper/get-secret.py -s {} -k {} -f {}'.format(
-                sentry_secret, 'dsn', '.sentry_token'
-            ),
+                secret_index, key, target_file
```

Comment:
The secrets (with multiple key/value pairs) needs to exist here:
https://tools.taskcluster.net/secrets

`project/mobile/reference-browser/sentry` exists. `project/mobile/reference-browser` is not a secret. Either we fetch multiple secrets here or we create a secret that holds multiple values. Let's just follow what Fenix does. :)